docs/api-reference/geo-layers/mvt-layer.md

```diff
@@ -161,6 +161,30 @@ new MVTLayer({
 })
 ```
 
+##### `onViewportLoad` (Function, optional)
```

Comment:
There is no need to re-document this prop from the base TileLayer class, since it behaves the same. Instead, add a `Tile` section to this doc to describe the additional field.